kalite/distributed/management/commands/screenshots.py

```diff
@@ -0,0 +1,338 @@
+                         interactive=False, stdout=new_io, router=router, verbosity=verbosity)
+            admin_user = User.objects.get(username=username)
+            admin_user.set_password(password)
+            admin_user.save()
+            return admin_user
+    return None
+
+
+def delete_sqlite_database(database=None, verbosity="1"):
+    """
+    Delete the specified sqlite database or if None, the one on `settings.py` of the app or project.
+    :param database: The database filename with full path.
+    """
+    try:
+        router = getattr(settings, 'SCREENSHOTS_ROUTER', 'default')
+        db_engine = settings.DATABASES[router]['ENGINE']
+        if db_engine == settings.SQLITE3_ENGINE:
+            if not database:
+                database = settings.DATABASES[router]['NAME']
+            if os.path.exists(database):
+                log.info('==> Removing database %s ...' % database) if int(verbosity) > 0 else None
+                os.remove(database)
+                log.info('====> Successfully removed database.') if int(verbosity) > 0 else None
+    except Exception as exc:
+        log.error('====> EXCEPTION: %s' % exc)
+
+
+class Screenshot(KALiteBrowserTestCase, FacilityMixins, BrowserActionMixins):
+    """
+    Override the values from base class for better looking screenshot data.
+    """
+    admin_username = 'admin'
+    admin_email = 'admin@example.com'
+    coach_username = 'coach'
+    student_username = 'student'
+    default_password = 'password'
+    facility = None
+
+    KEY_USERS = 'users'
+    KEY_SLUG = 'slug'
+    KEY_START_URL = 'start_url'
+    KEY_INPUTS = 'inputs'
+    KEY_PAGES = 'pages'
+    KEY_FOCUS = 'focus'
+    # Notes aren't used anywhere...
+    KEY_NOTES = 'notes'
+
+    KEY_CMD_SLUG = '<slug>'
+    KEY_CMD_SUBMIT = '<submit>'
+
+    JSON_KEYS = (KEY_USERS, KEY_SLUG, KEY_START_URL, KEY_INPUTS, KEY_PAGES, KEY_NOTES,)
+
+    database = getattr(settings, 'SCREENSHOTS_DATABASE_PATH', None)
+
+    verbosity = 1
+    output_path = None
+
+    def _fake_test():
+        # Fools django.utils.unittest.case.TestCase.__init__
+        # See the small novel comment under __init__ below
+        pass
+
+    def logwarn(self, *args, **kwargs):
+        if int(self.verbosity) > 0:
+            log.warn(*args, **kwargs)
+
+    def loginfo(self, *args, **kwargs):
+        if int(self.verbosity) > 0:
+            log.info(*args, **kwargs)
+
+    def __init__(self, *args, **kwargs):
+        # It's not good to override __init__ for classes that inherit from TestCase
+        # Since we're hackily inheriting here, we have to hackily invoke __init__
+        # Perhaps better would be to decouple this class from the testing framework
+        # by ditching the various mixins (they invoke TestCase methods) and just calling
+        # selenium methods directly, as the mixins are a thin wrapper for that.
+        # -- M.C. Gallaspy, 1/21/2015
```

Comment:
I believe we we considered using [splinter](https://splinter.readthedocs.org/en/latest/) as our selenium wrapper for the test infrastructure refactoring. If so, we can migrate this to use that instead of the browser mixins.